### PR TITLE
NavBar

### PR DESCRIPTION
**Pull request by mustiwebp**
This PR adds a nav bar

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,17 +1,34 @@
+// ignore_for_file: public_member_api_docs
+
+/*
+Copyright 2022 by Mustafa Sezer <mustafa.sezer@hit-solutions.de>
+
+This file is part of picos.
+
+picos is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
```
Comment (inchw0rm) [inline lib/main.dart:8]: Please format the license notice so that it does not exceed 80 characters per line.

File: lib/main.dart [hunk 2]
```diff
@@ -1,17 +1,34 @@
...
+Y; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.picos is distributed in the hope that it will be useful, but WITHOUT ANY WARRANT
+You should have received a copy of the GNU General Public License along with picos. If not, see <https://www.gnu.org/licenses/>.
+*/
+
 import 'package:flutter/material.dart';
 
 void main() {
-  runApp(const MyApp());
+  runApp(const Picos());
```
Comment (inchw0rm) [inline lib/main.dart:16]: I think this is a really unfortunate class name here, please use something more self-explainatory like MainAppScreen() or something you feel fits better.

File: lib/main.dart [hunk 3]
```diff
@@ -1,17 +1,34 @@
...
 }
 
-class MyApp extends StatelessWidget {
-  const MyApp({Key? key}) : super(key: key);
+class Picos extends StatefulWidget {
```
Comment (inchw0rm) [inline lib/main.dart:19]: Same goes with this class

File: lib/main.dart [hunk 4]
```diff
@@ -71,45 +102,64 @@ class _MyHomePageState extends State<MyHomePage> {
...
+      ),
+      backgroundColor: Colors.grey,
+      bottomNavigationBar: BottomNavigationBar(
+        items: const <BottomNavigationBarItem>[
+          BottomNavigationBarItem(
+              icon: Icon(
+                Icons.house_outlined,
+                color: Colors.black,
+              ),
+              label: 'Übersicht'),
```
Comment (inchw0rm) [inline lib/main.dart:123]: Please do not use hardcoded german labels.

File: lib/main.dart [hunk 5]
```diff
@@ -71,45 +102,64 @@ class _MyHomePageState extends State<MyHomePage> {
...
-        tooltip: 'Increment',
-        child: const Icon(Icons.add),
-      ), // This trailing comma makes auto-formatting nicer for build methods.
     );
   }
+
+  // ignore: always_specify_types
+  List<Widget> get appBarElements => [
+        Image.asset(
+          'assets/graphics/logo/appbar-logo.jpg',
```
Comment (inchw0rm) [inline lib/main.dart:153]: I am sorry I can't accept this PR due to this image.

**Review by inchw0rm — CHANGES_REQUESTED** [review]

File: lib/main.dart (hunk 4 above)
Comment (mustiwebp) [reply lib/main.dart:123]: Should I use constants instead?

Comment (inchw0rm) [reply lib/main.dart:123]: No, you should use proper localisation.
https://docs.flutter.dev/development/accessibility-and-localization/internationalization

File: lib/main.dart [hunk 6]
```diff
@@ -44,20 +61,34 @@ class MyHomePage extends StatefulWidget {
   final String title;
 
   @override
-  State<MyHomePage> createState() => _MyHomePageState();
+  State<MyHomePage> createState() => MainAppClass();
 }
 
-class _MyHomePageState extends State<MyHomePage> {
-  int _counter = 0;
+class MainAppClass extends State<MyHomePage> {
```
Comment (inchw0rm) [inline lib/main.dart:67]: Please add class annotations to every class you create.

**Comment by inchw0rm** [comment]
Sry you have still the image I asked you to remove in the history, please reset to a state where it has not been introduced and make a force push to your repository in order to remove it from the history as well.

**Pull request merged**